optimum/habana/diffusers/pipelines/pipeline_utils.py

```diff
@@ -370,6 +368,13 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
         bf16_full_eval = kwargs.get("torch_dtype", None) == torch.bfloat16
         kwargs["bf16_full_eval"] = bf16_full_eval
 
+        # Need to load custom ops lists before instantiating htcore
+        gaudi_config = GaudiConfig.from_pretrained(kwargs["gaudi_config"])
+        gaudi_config.declare_autocast_bf16_fp32_ops()
```

Comment:
```suggestion
        self.gaudi_config.declare_autocast_bf16_fp32_ops()
```

I think you could modify to this one. Unless you want to change the config in some cases from kwargs?